sound/soc/intel/boards/sof_sdw.c

```diff
@@ -583,6 +523,17 @@ static int create_codec_dai_name(struct device *dev,
 
 		codec[comp_index].dai_name =
 			codec_info_list[codec_index].dai_name;
+
+		codec_conf[*codec_conf_index].dlc = *codec;
+		codec_conf[*codec_conf_index].name_prefix = link->adr_d[i].name_prefix;
+
+		*codec_conf_index = *codec_conf_index + 1;
```

Comment:
++?

Reply:
I don't know my left from my right and I can never recall operator precedence. I e.g. don't know if *codec_conf_index++ will increment the value or the pointer.

Reply:
a trick: `++*p` is pretty sure to first dereference the pointer and then to increment the obtained variable :-) Another trick I use - I scanned the table with operator precedence from K&R and I keep it open most of the time, but please don't tell anyone, I don't want to be sued for copyright violation :-)

Reply:
right, thanks for the suggestion.